Tests/RegexTests/RenderDSLTests.swift

```diff
@@ -114,4 +114,12 @@ extension RenderDSLTests {
         """)
     }
   }
+  
+  func testQuoting() throws {
+    try testConversion(#"\\\"a\""#, #"""
+      Regex {
+        "\\\"a\""
```

Comment:
Is this the right conversion? The input is raw, so IIUC it has 3 full backslashes, then a quote then a full backslash and another quote.

Reply:
I'll update the test. The `\`s in `\"` in the regex to convert is ignored when parsing, but not necessary.